app/models/data_repo_client.rb

```diff
@@ -7,9 +7,13 @@ class DataRepoClient < Struct.new(:access_token, :api_root, :storage, :expires_a
   include ApiHelpers
 
   # Google authentication scopes necessary for querying TDR API
-  GOOGLE_SCOPES = %w(openid email profile)
+  GOOGLE_SCOPES = %w(openid email profile https://www.googleapis.com/auth/devstorage.read_only)
```

Comment:
how does this impact the scopes work from earlier this quarter?  Does this mean we will have to go back to prompting people again?

Reply:
No - this is the read-only service account we're generating the token for, so we can add/remove scopes as we see fit.  Since we don't plan on using people's credentials for accessing TDR, it doesn't require the usual OAuth "3 legged" authentication/confirmation flow.  In the future, if we did want to use user credentials, we can use the same [SAM endpoint](https://sam.dsde-prod.broadinstitute.org/#/Google/getArbitraryPetServiceAccountToken) for generating pet service account tokens with the `devstorage.read_only` scope, and would still not need to prompt users.